lib/scripts/windows/run-datadog.ps1

```diff
@@ -14,11 +14,16 @@ echo "$tags_out" | Out-File "$DATADOG_DIR\AppData\datadog.yaml" -Append -Encodin
 # Update the path to the check configuration files
 echo "confd_path: $DATADOG_DIR\AppData\datadog-agent\conf.d" | Out-File "$DATADOG_DIR\AppData\datadog.yaml" -Append -Encoding "UTF8"
 
-$Env:LOGS_CONFIG_DIR="$DATADOG_DIR\AppData\conf.d\logs.d"
-New-Item $Env:LOGS_CONFIG_DIR -ItemType "directory" -Force
+# Write the config for the logs
+$LOGS_CONFIG_DIR="$DATADOG_DIR\AppData\datadog-agent\conf.d\logs.d"
+New-Item $LOGS_CONFIG_DIR -ItemType "directory" -Force
+echo "{`"logs`":$Env:LOGS_CONFIG}" | Out-File $LOGS_CONFIG_DIR\logs.yaml
 
 # Logs, core checks, and dogstatsd
 Start-Process "$DATADOG_DIR\bin\agent.exe" -ArgumentList "run -c $DATADOG_DIR\AppData" -NoNewWindow
 
+# Start redirecting logs to the agent port
+Start-Process "$DATADOG_DIR\scripts\redirect_logs.ps1"
```

Comment:
Hmm that should not be here since we pipe the output from `hwc.exe` in the `run.cmd` (https://github.com/DataDog/datadog-cloudfoundry-buildpack/pull/82/files/f0d06788cc3f888914293b94b218c4d47c917370..88f63d7faae7b363fc415e8e64b73c3d1cc33d19#diff-04c6e90faac2675aa89e2176d2eec7d8R70)

Reply:
Good catch, this is leftover from a little while ago. Removing